.github/workflows/build_ubuntu.yml

```diff
@@ -0,0 +1,92 @@
+name: Build Ubuntu
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Info
+      run: echo GITHUB_WORKSPACE=$GITHUB_WORKSPACE
+      
+    - name: Apt update
+      run: sudo apt-get update
+
+    - name: Install tools
+      run: sudo apt-get -y install git cmake build-essential gettext help2man openssl wget #clang-3.8
+ 
+    - name: Prepare sources for act (run actions local)
+      run: echo #rm -r 3rdparty/aubio 3rdparty/ced
+
+    - name: Checkout 
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: Prepare Submodules
+      run : git submodule update --init --recursive
+
+    - name: Install libraries
+      run: |
+        #sudo apt-get -y install software-properties-common
+        sudo apt-get -y install libblas-dev #liblapack-dev
+        sudo apt-get -y install fftw3-dev fftw3
+        sudo apt-get -y install libicu-dev 
+        sudo apt-get -y install libepoxy-dev 
+        sudo apt-get -y install libsdl2-dev 
+        sudo apt-get -y install libfreetype6-dev
+        sudo apt-get -y install libpango1.0-dev
+        sudo apt-get -y install librsvg2-dev
+        sudo apt-get -y install libxml++2.6-dev 
+        sudo apt-get -y install libavcodec-dev libavformat-dev libswscale-dev libswresample-dev libjpeg-dev
+        sudo apt-get -y install portaudio19-dev 
+        sudo apt-get -y install libboost-all-dev
+        sudo apt-get -y install libglm-dev 
+        #sudo apt-get -y install libopencv-dev 
+        #sudo apt-get -y install libportmidi-dev 
+        #sudo apt-get -y install libssl-dev 
```

Comment:
All `sudo apt-get -y install` can be combined into one big statement. I'd also recommend disabling the recommended installs with `--no-install-recommends`

Also please enable all packages.